
uint8 VEGATATION_NONE = 0
uint8 VEGATATION_GRASS = 1
uint8 VEGATATION_SHRUB = 2
uint8 VEGATATION_FLOWER = 3


uint8 rubbish_grade
bool has_person
uint8 vegetation_type
float32 max_volumn
float32 total_weight
uint8 area_id
